docs: fix build output path in App Shell guide

hadrien-toma opened the pull request:
Adding the App Shell prefixes the dist output folder by `browser`, so the build output path in the guide should to reflect it.

## PR Checklist

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

hadrien-toma committed:
docs: fix build output path in App Shell guide

Adding the App Shell prefixes the dist output folder by `browser`, so the build output path in the guide should to reflect it.

hadrien-toma committed:
docs(guide/app-shell.md): fix build output path

Adding the App Shell prefixes the dist output folder by `browser`, so the build output path in the guide should to reflect it.

hadrien-toma commented:
@googlebot I signed it!

gkalpak reviewed (changes requested):
Good catch! Thx, @hadrien-toma :+1:

There are couple of things that need fixing:

1. The commit message does not exactly follow our [commit message guidelines][1]. It should be something like:
    > docs: fix build output path in App Shell guide
    >
    > Adding the App Shell ...

2. The [`server` config][2] in `angular.json` seems outdated:
    ```diff
     "server": {
       "builder": "@angular-devkit/build-angular:server",
       "options": {
    -    "outputPath": "dist/my-app-server",
    +    "outputPath": "dist/my-app/server",
         "main": "src/main.server.ts",
         "tsConfig": "tsconfig.server.json"
    +  },
    +  "configurations": {
    +    "production": {
    +      "outputHashing": "media",
    +      "fileReplacements": [
    +        {
    +          "replace": "src/environments/environment.ts",
    +          "with": "src/environments/environment.prod.ts"
    +        }
    +      ],
    +      "sourceMap": false,
    +      "optimization": true
    +    }
       }
     },
    ```

[1]: https://github.com/angular/angular/blob/1d9e56eba06958e37ef8062ae7b65afa85f90632/CONTRIBUTING.md#commit
[2]: https://github.com/angular/angular/blame/ba45a8e443af2ca0e053e47579d6d667d5bc75e6/aio/content/guide/app-shell.md#L32-L39

hadrien-toma committed:
docs: update angular.json quote in App Shell guide

The angular.json quoted in the App Shell guide needed to be updated to reflect the actual behavior induced by the app-shell schematic.

hadrien-toma committed:
docs: update angular.json quote in App Shell guide

The angular.json quoted in the App Shell guide needed to be updated to reflect the actual behavior induced by the app-shell schematic.

hadrien-toma commented:
Hey @gkalpak, thanks for your feedbacks, I tried to be compliant according to them. Is it now correct? :blush:

hadrien-toma commented:
So it seems its not okay because I can't find how to remove the 4 commits and replace them by only one, properly formatted, can you guide me in this process? Should I remove this PR and create another one?

hadrien-toma commented:
I will do this, I am creating another one, it is easy and quick.

hadrien-toma commented:
Replaced by https://github.com/angular/angular/pull/40210
